example_profile/example_config.yaml

```diff
@@ -534,6 +534,44 @@ HEATMAP_FEATURE_CORRELATION_MATRIX:
   CORR_METHOD: "pearson" # choose from {"pearson", "kendall", "spearman"}
 
 
+########################################################################################################################
+#                                                    Data Cleaning                                                     #
+########################################################################################################################
+
+ALL_CLEANING_INDIVIDUAL:
+  PROVIDERS:
+    RAPIDS:
+      COMPUTE: True
+      IMPUTE_SELECTED_EVENT_FEATURES:
+        COMPUTE: False
+        MIN_DATA_YIELDED_MINUTES_TO_IMPUTE: 0.33
+      COLS_NAN_THRESHOLD: 0.3 # set to 1 to disable
+      COLS_VAR_THRESHOLD: True
+      ROWS_NAN_THRESHOLD: 0.3 # set to 1 to disable
+      DATA_YIELDED_HOURS_RATIO_THRESHOLD: 0.75 # set to 0 to disable
+      DROP_HIGHLY_CORRELATED_FEATURES:
+        COMPUTE: False
+        MIN_OVERLAP_FOR_CORR_THRESHOLD: 0.5
+        CORR_THRESHOLD: 0.95
+      SRC_SCRIPT: src/features/all_cleaning_individual/rapids/main.R
+
+ALL_CLEANING_OVERALL:
```

Comment:
Did we explain the difference between individual & overall in the docs?

Reply:
I think so. We have two sections in the docs: 
(1) Clean sensor features for individual participants; 
(2) Clean sensor features for all participants.

The only difference between those two sections is the input file. I listed the "File Sequence" for both sections to denote different input files: https://github.com/carissalow/rapids/blob/data_cleaning/docs/analysis/data-cleaning.md?plain=1#L68

Hi @JulioV, is it ok? Do I need to add descriptions at the beginning? For example, line4:
The goal of this module is to perform basic clean tasks on the behavioral features that RAPIDS computes. You might need to do further processing depending on your analysis objectives. This module can clean sensor features for both individual participants and all participants. You need to set parameters within each section (e.g. `[ALL_CLEANING_INDIVIDUAL]` and `[ALL_CLEANING_OVERALL]`).

Reply:
Yeah let's add this. Small correction, this module can clean features at the individual level and at the study level. And give an example of why this is important please

Reply:
Thanks @JulioV. How about this example:

This module can clean features at the individual level and at the study level. For example, if we have enough data for a participant, we can train a model for that participant. By setting parameters in `[ALL_CLEANING_INDIVIDUAL]` section, we can get cleaned data for that participant. Similarly, if we want to train a model for all participants, we can get the cleaned data for all participants by setting parameters in `[ALL_CLEANING_OVERALL]` section.

Reply:
This module can clean features at the individual level and at the study level. If you are interested in creating individual models (using each participant's features independently of the others) use `ALL_CLEANING_INDIVIDUAL`. If you are interested in creating population models (using everyones data in the same model) use `ALL_CLEANING_OVERALL`

I think this is clearer. What do you think?

Reply:
Yes! Thank you!